### PR TITLE
Test upgrade access_controlable

### PR DESCRIPTION
**Pull request by kiseln**
A test that verifies that `access_controlable` contract changes are backwards compatible

### CODE REVIEW

File: near-plugins-derive/tests/access_controllable.rs [hunk 1]
```diff
@@ -1520,3 +1534,41 @@ async fn test_acl_revoke_role_unchecked_is_private() -> anyhow::Result<()> {
...
+    let role = "ByMax2Increaser";
+
+    let old_wasm = std::fs::read(WASM_V_0_2_0_FILEPATH)?;
+    let Setup {
+        contract, account, ..
+    } = Setup::new_with_wasm(old_wasm).await?;
+
+    let contract_account = contract.contract().as_account();
+    let _ = contract
+        .acl_grant_role_unchecked(contract_account, role, account.id())
```
Comment (karim-en) [inline near-plugins-derive/tests/access_controllable.rs:1551]: Let's add more storage changes

File: near-plugins-derive/tests/access_controllable.rs [hunk 2]
```diff
@@ -1520,3 +1534,41 @@ async fn test_acl_revoke_role_unchecked_is_private() -> anyhow::Result<()> {
...
+async fn test_upgrade_storage() -> anyhow::Result<()> {
+    let role = "ByMax2Increaser";
+
+    let old_wasm = std::fs::read(WASM_V_0_2_0_FILEPATH)?;
+    let Setup {
+        contract, account, ..
+    } = Setup::new_with_wasm(old_wasm).await?;
+
+    let contract_account = contract.contract().as_account();
+    let _ = contract
```
Comment (karim-en) [inline near-plugins-derive/tests/access_controllable.rs:1550]: Can we check the result?

**Review by karim-en — CHANGES_REQUESTED** [review]

File: near-plugins-derive/tests/access_controllable.rs (hunk 1 above)
Comment (kiseln) [reply near-plugins-derive/tests/access_controllable.rs]: Added some more

Comment (karim-en) [reply near-plugins-derive/tests/access_controllable.rs]: Can you please also add checks for these roles/changes

Comment (kiseln) [reply near-plugins-derive/tests/access_controllable.rs]: Done

File: near-plugins-derive/tests/data/access_controllable_v_0_2_0.wasm
Comment (karim-en) [inline near-plugins-derive/tests/data/access_controllable_v_0_2_0.wasm:1]: The contract binary size is huge, can you please try to build it using this flags. 

RUSTFLAGS='-C link-arg=-s' cargo build --target wasm32-unknown-unknown --release

More details 
https://docs.near.org/build/smart-contracts/anatomy/reduce-size

**Review by karim-en — CHANGES_REQUESTED** [review]

File: near-plugins-derive/tests/data/access_controllable_v_0_2_0.wasm
Comment (kiseln) [reply near-plugins-derive/tests/data/access_controllable_v_0_2_0.wasm:1]: Done

**Review by karim-en — APPROVED** [review]

**Pull request merged**